drbd/initial_sync.sls

```diff
@@ -4,7 +4,7 @@
 {% for res in drbd.resource %}
 {% if drbd.need_format is defined and drbd.need_format is sameas true%}
 {% if res.file_system == 'xfs' %}
-init_drbd_install_xfs:
+init_drbd_install_xfs-{{ res.name }}:
```

Comment:
This doesn't make much sense.
If the issue is that due the multiple loops, the id gets duplicated, you could maybe break after installation is done.
I don't know if the `break` option is available in jinja though

Reply:
You are right.